Jenkinsfile

```diff
@@ -629,10 +618,10 @@ pipeline {
   }
 
   triggers {
-    parameterizedCron((scheduleTriggerJobName() == env.JOB_NAME) ? """
+    parameterizedCron(branchPatternCron().matcher(env.BRANCH_NAME).matches() ? """
       # Every weeknight (Monday - Friday) around 4:00 AM
-      # These schedules will run with and without Cython enabled for Python v2.7.18 and v3.5.9
-      H 4 * * 1-5 %CI_SCHEDULE=WEEKNIGHTS;EVENT_LOOP=LIBEV;CI_SCHEDULE_PYTHON_VERSION=2.7.18 3.5.9;CI_SCHEDULE_SERVER_VERSION=2.2 3.11 dse-5.1.35 dse-6.0.18 dse-6.7.17
+      # These schedules will run with and without Cython enabled for Python 3.7.7 and 3.8.3
+      H 4 * * 1-5 %CI_SCHEDULE=WEEKNIGHTS;EVENT_LOOP=LIBEV;CI_SCHEDULE_PYTHON_VERSION=3.7.7 3.8.3;CI_SCHEDULE_SERVER_VERSION=2.2 3.11 dse-5.1.35 dse-6.0.18 dse-6.7.17
```

Comment:
Part of my goal here is to get nightlies working again.  PYTHON-1351 will need some environment customizations to make sure we can test the cryptography and non-cryptography cases.  It probably makes sense to include one as part of the default build and the other in the nightlies.

Reply:
This probably won't be as bad as it looked initially.  PYTHON-1351 is following the example of similar functionality in the Python driver (for a Gremlin dependency for DS Graph) and importing functionality specific to cryptography only if the lib is present.  Since cryptography will be present in the Jenkins environment [1]  those tests will be run.

[1] cryptography has now been moved to test-datastax-requirements.txt, the same spot where the Gremlin requirement lives.  Requirements in that file are imported when the Jenkins environment is setup.